docs/source/support/release-notes.rst

```diff
@@ -76,14 +83,66 @@ Java Bindings
 
   See `issue #1092 <https://github.com/digital-asset/daml/issues/1092>`__ for details.
 
+- Added agreement text of contracts: `#1110 <https://github.com/digital-asset/daml/issues/1110>`__
+
+  - **Java Bindings**
+
+    - Added field ``Optional<String> agreementText`` to ``data.CreatedEvent``, to reflect the change in Ledger API.
+
+  - **Java Codegen**
+
+    - Added generated field ``Optional<String> TemplateName.Contract#agreementText``.
+    - Added generated static method ``TemplateName.Contract.fromCreatedEvent(CreatedEvent)``.
+    - Added generated static method ``TemplateName.Contract.fromIdAndRecord(String, Record, Optional<String>)``.
+    - Deprecated generated static method ``TemplateName.Contract.fromIdAndRecord(String, Record)`` in favor of the new static methods in the generated ``Contract`` classes.
+    - Changed the generated `decoder utility class <https://docs.daml.com/app-dev/bindings-java/codegen.html#generate-the-decoder-utility-class>`__ to use the new ``fromCreatedEvent`` method.
+    - Changed the return type of the ``getDecoder`` method in the generated decoder utility class from ``Optional<BiFunction<String, Record, Contract>>`` to ``Optional<Function<CreatedEvent, Contract>>``.
+
+  How to migrate:
```

Comment:
If there are migration instructions, does that mean it's a breaking change?

Reply:
I'll mark this change as breaking.

Reply:
See 11a465f4.